doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -118,10 +130,10 @@ Other tokens
   without using :cmd:`Notation` (omitting 25 specialized tokens that begin with
   ``#int63_``)::
 
-    ! #[ % & ' ( () (bfs) (dfs) ) * ** + , - ->
+    ! #[ % & ' ( () ) * + , - ->
     . .( .. ... / : ::= := :> :>> ; < <+ <- <:
-    <<: <= = => > >-> >= ? @ @{ [ [= ] _ _eqn
-    `( `{ { {| | |- || }
+    <<: <= = => > >-> >= ? @ @{ [ ] _
```

Comment:
No grammar change in order to remove || as a token?  They still appear in the grammar and I think they're still tokens.

Reply:
Hugo moved the mention of this token to the Ltac chapter. @herbelin Does this mean that `||` is not a token if Ltac isn't loaded (e.g. with `-nois`)?

Reply:
Yes, `||` is not in the initial state.

Reply:
Then, moving it to the Ltac chapter is the right thing to do.

Reply:
But also used in SSR, maybe should be mentioned in both.